src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,19 @@
+package duke.task;
+
+public class Event extends Task {
+    protected String time;
+
```

Comment:
Maybe this could be named at to follow the same naming format used in the Deadline class?